edgelet/support-bundle/src/support_bundle.rs

```diff
@@ -1,5 +1,16 @@
 // Copyright (c) Microsoft. All rights reserved.
 
+// TODO:
```

Comment:
ehh, I feel like it's probably fine lumping this TODO in with the rest of these changes. You know as well as I do that the moment you put a TODO like this into the codebase, unless we actually make a work-item for it, it'll just linger for a long while...

Reply:
Would the replacement for this be tokio 0.1 compatible? If not, it isn't worth the switch now maybe. 

---
In reply to: [544641313](https://github.com/Azure/iotedge/pull/4106#discussion_r544641313) [](ancestors = 544641313)

Reply:
:effort:, but fine. Done.